akka-docs/src/main/paradox/additional/rolling-updates.md

```diff
@@ -74,3 +80,68 @@ and ensure all nodes are in this state
+      make one change at a time.
+1. Rolling update to support deserialization but not enable serialization
+    * Change message classes by adding the marker interface and possibly needed annotations as
+      described in @ref:[Serialization with Jackson](../serialization-jackson.md).
+    * Test the system with the new serialization in a new test cluster (no rolling update).
+    * Remove the binding for the marker interface, so that Jackson is not used for serialization yet.
+    * Roll out the change.
+    * Java serialization is still used, but this version is prepared for next roll out.
+1. Rolling update to enable serialization with Jackson.
+    * Add the binding to the marker interface to the Jackson serializer.
+    * Roll out the change.
+    * Old nodes will still send messages with Java serialization, and that can still be deserialized by new nodes.
+    * New nodes will send messages with Jackson serialization, and old node can deserialize those because they were
+      prepared in previous roll out.
+1. Rolling update to disable Java serialization
+    * Remove `allow-java-serialization` config, to use the default `allow-java-serialization=off`.
+    * Remove `.warn-about-java-serializer-usage` config if you had changed that, to use the default `.warn-about-java-serializer-usage=on`.
+    * Roll out the change.
+    
+A similar approach can be used when changing between other serializers, for example between Jackson and Protobuf.    
+
+## When Shutdown Startup Is Required
+ 
+There are a few instances when a full shutdown and startup is required versus being able to do a rolling update.
+
+### Cluster Sharding configuration change
+
+If you need to change any of the following aspects of sharding it will require a full cluster restart versus a rolling update:
+
+ * The `extractShardId` function
+ * The role that the shard regions run on
+ * The persistence mode - It's important to use the same mode on all nodes in the cluster 
+ 
+### Migrating from PersistentFSM to EventSourcedBehavior
+
+If you've [migrated from `PersistentFSM` to `EventSourcedBehavior`](../persistence-fsm.md#migration-to-eventsourcedbehavior)
+and are using PersistenceFSM with Cluster Sharding, a full shutdown is required as shards can move between new and old nodes.
+  
+### Migrating from classic remoting to Artery
+
+If you've migrated from classic remoting to Artery
+which has a completely different protocol, a rolling update is not supported.
+For more details on this migration
+see @ref:[the migration guide](../project/migration-guide-2.5.x-2.6.x.md#migrating-from-classic-remoting-to-artery).
+
+### Akka Typed with Receptionist or Cluster Receptionist
+
+If you are migrating from Akka 2.5 to 2.6, using the `Receptionist` or `Cluster Receptionist` with Akka Typed, information will not be disseminated between 2.5 and 2.6 nodes during a
+rolling update from 2.5 to 2.6. When all old nodes have been shutdown it will work properly again.
```

Comment:
I still don't think this belong in the section "When Shutdown Startup Is Required". What was the reason for having it there when it work "good enough" also for a rolling update?

Reply:
ok, "all old nodes have been shutdown" refers to when all old nodes have been phased out during the rolling update. Pls find better wording.